style.css

```diff
@@ -0,0 +1,9 @@
+#navbar{
+    position:fixed;
+    top:0;
+}
+@media only screen and (min-width: 500px) {
+body {
+  background-color:tomato;
+}
+}
```

Comment:
Nice job coding this! 👍. Remember to always add indentation where it is needed. Your body selector and all the elements it holds should have an indentation to show it is inside of the '@media' query. Having good indentation will make it easier for anyone to read and make changes to your code. 👀